dataikuapi/dss/admin.py

```diff
@@ -795,9 +795,53 @@ def set_definition(self, definition):
             definition: the definition for the API key, as a JSON object.                        
         """
         return self.client._perform_empty(
-            "PUT", "/admin/globalAPIKeys/%s" % self.key,
+            "PUT", "/admin/global-api-keys/%s" % self.id,
             body = definition)
 
+class DSSPersonalApiKey(object):
+    """
+    A personal API key on the DSS instance
+    """
+    def __init__(self, client, id):
+        self.client = client
+        self.id = id
+
+    ########################################################
+    # Key deletion
+    ########################################################
+
+    def delete(self):
+        """
+        Delete the api key
+        """
+        return self.client._perform_empty(
+            "DELETE", "/personal-api-keys/%s" % self.id)
+
+    ########################################################
+    # Key description
+    ########################################################
+
+    def get_definition(self):
+        """
+        Get the API key's definition
+
+        Returns:
+            the personal API key definition, as a JSON object
+        """
+        return self.client._perform_json(
+            "GET", "/personal-api-keys/%s" % (self.id))
+
+    def update(self, label="", description=""):
+        """
+        Get the API key's definition
+        :param: label string (optional): Label for the API key
+        :param: description string (optional): Description for the API key
```

Comment:
Here you need something like `:param str label: Label for the API key (optional)` and also a empty line between this first param and the `Get the API key's definition` line. Similar for `create_personal_api_key` below.